MapboxVision/src/main/java/com/mapbox/vision/utils/FileUtils.kt

```diff
@@ -34,4 +34,10 @@ internal object FileUtils {
 
         return "${getAbsoluteDir(dir.absolutePath)}/"
     }
+
+    fun moveFilesFromOneDirToAnother(fromDirPath: String, toDirPath: String) {
```

Comment:
Use `renameTo` for consistency with `File` api?

Reply:
refactor